.github/workflows/smoke-test-daily.yml

```diff
@@ -52,3 +52,103 @@ jobs:
+          with:
+              name: woocommerce
+              path: ${{ steps.build.outputs.zip_path }}
+              retention-days: 7
+    
+  test-plugins:
+    name: Smoke tests with ${{ matrix.plugin }} plugin installed
+    runs-on: ubuntu-18.04
+    needs: [build]
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - plugin: 'WooCommerce Payments'
+            repo: 'automattic/woocommerce-payments'
+          - plugin: 'Facebook for WooCommerce'
+            repo: 'woocommerce/facebook-for-woocommerce'
+          - plugin: 'WooCommerce Shipping & Tax'
+            repo: 'woocommerce/woocommerce-services'
+          - plugin: 'WooCommerce Subscriptions'
+            repo: WC_SUBSCRIPTIONS_REPO
+            private: true
+          - plugin: 'WordPress SEO' # Yoast SEO in the UI, but the slug is wordpress-seo
+            repo: 'Yoast/wordpress-seo'
+          - plugin: 'Contact Form 7'
+            repo: 'takayukister/contact-form-7'
+    steps:
+      - name: Create dirs.
+        run: |
+            mkdir -p code/woocommerce
+            mkdir -p package/woocommerce
+            mkdir -p tmp/woocommerce
+            mkdir -p node_modules
+
+      - name: Checkout code.
+        uses: actions/checkout@v2
+        with:
+            path: package/woocommerce
+
+      - name: Install PNPM and install dependencies
+        working-directory: package/woocommerce
+        run: |
+            npm install -g pnpm
+            pnpm install
+
+      - name: Load docker images and start containers.
+        working-directory: package/woocommerce/plugins/woocommerce
+        run: pnpx wc-e2e docker:up
+
+      - name: Move current directory to code. We will install zip file in this dir later.
+        run: mv ./package/woocommerce/plugins/woocommerce/* ./code/woocommerce
+
+      - name: Download WooCommerce ZIP.
+        uses: actions/download-artifact@v2
+        with:
+            name: woocommerce
+            path: tmp
+
+      - name: Extract and replace WooCommerce zip.
+        working-directory: tmp
+        run: |
+            unzip woocommerce.zip -d woocommerce
+            mv woocommerce/woocommerce/* ../package/woocommerce/plugins/woocommerce/
+
+      - name: Install dependencies again
+        working-directory: package/woocommerce
+        run: |
+            npm install -g pnpm
+            pnpm install
+
+      - name: Run tests command.
+        working-directory: package/woocommerce/plugins/woocommerce
+        env:
+          WC_E2E_SCREENSHOTS: 1
+          E2E_SLACK_TOKEN: ${{ secrets.SMOKE_TEST_SLACK_TOKEN }}
+          E2E_SLACK_CHANNEL: ${{ secrets.RELEASE_TEST_SLACK_CHANNEL }}
+          GITHUB_REPOSITORY: ${{ matrix.private && secrets[matrix.repo] || matrix.repo }}
+          PLUGIN_NAME: ${{ matrix.plugin }}
+          GITHUB_TOKEN: ${{ secrets.E2E_GH_TOKEN }}
+        run: |
+          pnpx wc-e2e test:e2e plugins/woocommerce/tests/e2e/specs/smoke-tests/upload-plugin.js
```

Comment:
We will leave this for now but we may need to remove the duplicate `plugins/woocommerce` in a follow up PR.

Reply:
Added a follow up issue for this here: https://github.com/woocommerce/woocommerce/issues/31467